tools/vim/.ycm_extra_conf.py

```diff
@@ -0,0 +1,106 @@
+# A modified example from YCM main repository
```

Comment:
Uhh. IANAL. We have the requirement that things in this repository _have_ to be under the Boost license. I'd rather not have something in the repository if we can't reliably rule out licensing issues.